test/Microsoft.AspNetCore.Mvc.FunctionalTests/BasicApiTest.cs

```diff
@@ -56,7 +57,9 @@ public BasicApiTest(BasicApiFixture fixture)
             Assert.Equal(HttpStatusCode.Forbidden, response.StatusCode);
         }
 
-        [Fact]
+        // Tests are conditional to avoid occasional CI failures on Windows 8 and Windows 2012 under full framework.
+        [ConditionalFact]
+        [FrameworkSkipCondition(RuntimeFrameworks.CLR, SkipReason = "See aspnet/Identity#1630")]
```

Comment:
I'll also add a comment in aspnet/Identity#1630 pointing to this commit (once it's final)